packages/server/utils/analytics/analytics.ts

```diff
@@ -317,7 +317,7 @@ class Analytics {
   }
 
   toggleSubToSummaryEmail = (userId: string, subscribeToSummaryEmail: boolean) => {
-    this.track(userId, 'Summary Email Setting Changed', subscribeToSummaryEmail)
+    this.track(userId, 'Summary Email Setting Changed', {subscribeToSummaryEmail})
   }
 
   private track = (userId: string, event: AnalyticsEvent, properties?: any) =>
```

Comment:
-1 If you see a type mismatch bug like this, you should always ask yourself if this could have been catched by typescript. In this case it could have been if the typing here would be a bit more strict.
```ts
  private track = (userId: string, event: AnalyticsEvent, properties?: Record<string, any>) =>
```

Reply:
I agree with Georg, let's take this chance and improve this!

Reply:
Thanks, good call - updated!